test/analysis/test_utils.py

```diff
@@ -47,6 +49,50 @@ def test_to_int_label_array(self):
     def test_pred_to_prob(self):
         np.testing.assert_array_equal(preds_to_probs(PREDS, 2), PREDS_ROUND)
 
+    def test_hash(self):
```

Comment:
Don't think we need this...not testing an actual property of the implementation. Deterministic-ness is tested in test_break_ties.